entrypoint.sh

```diff
@@ -613,6 +617,12 @@ prepare() { local mode no_tap=1
 		no_tap=0 # we want subtests
 	fi
 
+	cat <<EOF > "${BASH_PROFILE}"
+	export KERNEL_BUILD_DIR="${VIRTME_BUILD_DIR}"
```

Comment:
while at it, feel free to also export `KERNEL_SRC_DIR`, it might be helpful later.